lib/assesments/assessment_2/fi_a2_muhammad_bilal/fi_a2_mbilal_homescreen/widgets/fi_a2_mbilal_homecontainer.dart

```diff
@@ -0,0 +1,125 @@
+
+import 'package:flutter/material.dart';
+import 'package:flutter_svg/svg.dart';
+
+class BilalHomeContainer extends StatelessWidget {
```

Comment:
Please name the class properly, HomeContainer does not describe the purpose of this